Tema-2/script1.js

```diff
@@ -0,0 +1,15 @@
+console.log("Exercitiul 1");
+
+let numar=123;
+let rest=numar % 2;
+ 
+if(rest=0){
```

Comment:
ai folosit = in loc de ===

Reply:
Am modificat aici demult, mie imi apare cum trebuie